installing/installing_openstack/installing-openstack-installer-restricted.adoc

```diff
@@ -34,7 +34,7 @@ include::modules/installation-osp-bootstrap-machine.adoc[leveloffset=+2]
 include::modules/cluster-entitlements.adoc[leveloffset=+1]
 include::modules/installation-osp-enabling-swift.adoc[leveloffset=+1]
 include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
-include::modules/installation-osp-creating-image-restricted.adoc[leveloffset=+1]
+include::modules/installation-creating-image-restricted.adoc[leveloffset=+1]
```

Comment:
Just to check, is this the same content as `modules/installation-osp-creating-image-restricted.adoc` and just updates the module file name?

Reply:
That's correct. Since I was reusing the module elsewhere, I made the file name generic.